config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  resources :snapshots, :only => [:index]
-  resources :sites, :only => [:index]
-  get 'trending', to: 'stories#trending'
-  get 'impact', to: 'stories#impact'
+  get "impact", to: "stories#impact"
+  get "sites", to: "sites#index"
+  get "snapshots", to: "snapshots#index"
```

Comment:
Figured that since we're only using `#index` anyway, may as well make the route explicit, since `resources` usually indicates more than one action.